src/Microsoft.Health.Dicom.Api/Controllers/ExtendedQueryTagController.cs

```diff
@@ -132,6 +132,35 @@ public async Task<IActionResult> GetTagAsync(string tagPath)
                 (int)HttpStatusCode.OK, response.ExtendedQueryTag);
         }
 
+        /// <summary>
+        /// Handles requests to get extended query tag errors.
+        /// </summary>
+        /// <param name="tagPath">Path for requested extended query tag.</param>
+        /// <returns>
+        /// Returns Bad Request if given path can't be parsed. Returns Not Found if given path doesn't map to a stored
+        /// error. Returns OK with a JSON body of requested tag error in other cases.
+        /// </returns>
+        [ProducesResponseType(typeof(GetExtendedQueryTagEntry), (int)HttpStatusCode.OK)]
+        [ProducesResponseType(typeof(string), (int)HttpStatusCode.BadRequest)]
+        [ProducesResponseType(typeof(string), (int)HttpStatusCode.NotFound)]
+        [HttpGet]
+        [VersionedRoute(KnownRoutes.GetExtendedQueryTagErrorsRoute)]
+        [Route(KnownRoutes.GetExtendedQueryTagErrorsRoute)]
+        [AuditEventType(AuditEventSubType.GetExtendedQueryTag)]
+        public StatusCodeResult GetTagErrorAsync(string tagPath)
+        {
+            _logger.LogInformation("DICOM Web Get Extended Query Tag Errors request received for extended query tag: {tagPath}", tagPath);
+
+            EnsureFeatureIsEnabled();
+            // TODO: Implement Service method.
```

Comment:
I think you can go so far as to define the interfaces/services leveraged by this controller